packages/core/admin/admin/src/pages/SettingsPage/pages/TransferTokens/EditView/reducer.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable consistent-return */
+import produce from 'immer';
+import { pull } from 'lodash';
+import { transformPermissionsData } from './utils';
+
+export const initialState = {
+  data: {},
+  selectedActions: [],
+};
+
+const reducer = (state, action) =>
```

Comment:
same here those are not needed

Reply:
yes, thanks Bassel, we have removed the file